project-base/app/config/packages/frontend_api.yaml

```diff
@@ -0,0 +1,5 @@
+parameters:
+    shopsys.frontend_api.keys_filepath: '%kernel.project_dir%/config/frontend-api'
+    shopsys.frontend_api.domains:
+        - 1
+        - 2
```

Comment:
In that case, you should also remove calling `php phing frontend-api-enable` from CI scripts